koocook_core/management/commands/_add_path.py

```diff
@@ -0,0 +1,3 @@
+def add_datatrans():
+    import sys
+    sys.path.insert(0, 'C:\\Users\\User\\PycharmProjects\\datatrans')
```

Comment:
Again, this is OS-specific path. Another workaround on this?

Reply:
You'd have to get `datatrans` in there somehow. IDK how to best do that. You tell me? `.env` file?

Reply:
Resolved in a24169